safe-global/safe-wallet-web
# feat: show message and domain hash

## What it solves

Resolves #4391 

## How this PR fixes it
Shows the internal message and domain hashes next to the safeTxHash.

## How to test it
- Create any Safe tx
- Observe the new hashes in the Advanced details
- Compare them with e.g. the message and domain hash a Ledger Nano S displays.

## Screenshots
![Screenshot 2024-10-18 at 15 55 54](https://github.com/user-attachments/assets/ee2cdfe8-4c84-4f1c-bcb7-ea3100f890b2)

## Checklist
